### ISSUE TITLE
[Detection Engine] - Alert tagging & close reason

### ISSUE THREAD

**Issue by yctercero** [elastic/security-docs#3263]
## Description 

The alert tagging feature allows security analysts to provide descriptive reasons for why they closed alerts. Analysts can manage tags from the Alerts table or from within the alert details flyout. They can then filter tagged alerts by searching for the [`kibana.alert.workflow_tags`](https://www.elastic.co/guide/en/security/8.8/alert-schema.html#alert-schema) field (e.g., `kibana.alert.workflow_tags : "False Positive" `) and display tags in the Alerts table by adding the `kibana.alert.workflow_tags` field. 

Related: 
- https://github.com/elastic/kibana/pull/157786
- https://github.com/elastic/security-team/issues/3273
- https://github.com/elastic/kibana/pull/160305: Updates copy for the option and menu.

### Doc updates 
- [x] [Take actions on alerts](https://www.elastic.co/guide/en/security/master/alerts-ui-manage.html#alert-actions): Add a new section for managing (adding and deleting) alert tags to individual and multiple alerts.
- [x] [Alert schema](https://www.elastic.co/guide/en/security/8.8/alert-schema.html#alert-schema): Add a new entry for the new `kibana.alert.workflow_tags` field. 
- [x] [Signals endpoint API](https://www.elastic.co/guide/en/security/master/signals-api-overview.html#signals-api-overview): Add a new section for managing (adding and deleting) alert tags. 
- [x] [Configure advanced settings](https://www.elastic.co/guide/en/security/8.8/advanced-settings.html): Describe the new 	`securitySolution:alertTags` setting.
- [x] Add front matter to all topics.

## Notes  

- Users can add tags to individual or multiple alerts.
- Tag options are generated from a field in the advanced settings and will be defaulted to 3 options: 
  - Duplicate
  - False positive
  - Further investigation required
- The `kibana.alert.workflow_tags` field was created to avoid conflicts with the source event tags that are added to documents (see `tag` field description [here](https://www.elastic.co/guide/en/security/master/alert-schema.html)).
- Users can _add and remove_ tags to alerts via the following route: `POST <kibana host>:<port>/api/detection_engine/signals/tags`. They can set the following parameters:
````
{
  tags: {
    tags_to_add: string[],
    tags_to_remove: string[]
  },
  ids: string[]
}
````
- [x] Get sample request and response/payload from @dplumlee

**Comment by nastasha-solomon** [comment]
Note to self: Need to follow-up with @dplumlee to verify if this will be shipped in 8.9.

**Comment by yctercero** [comment]
@nastasha-solomon it made it in before feature freeze! So it's a go.
